### PR TITLE
Career page ++

### PR DESCRIPTION
**Pull request by Woldt**

### CODE REVIEW

File: web/src/components/page/calculator.module.css [hunk 1]
```diff
@@ -1,13 +1,12 @@
-.calculator {
-  max-width: 550px;
-  width: 100%;
-}
```
Comment (andrnors) [inline web/src/components/page/calculator.module.css:1]: Can we rename this file to ```Calculator.module.css```, so it corresponds to the main component?

File: web/src/pages/career/index.tsx [hunk 2]
```diff
@@ -1,112 +1,30 @@
...
 interface Props {
-  jobs: Array<JobDescription>;
+  page: NonNullable<Sanity.Default.Schema.Page>;
 }
-
-const CareerPage: NextPageWithLayout = ({ jobs }: Props) => {
+export default function NewCareerPage({ page }: Props) {
   usePageTitle('Career');
 
+  // TODO We can add section for calculator and benefits under page builder
```
Comment (andrnors) [inline web/src/pages/career/index.tsx:12]: Is this TODO resolved?

File: web/src/pages/projects/index.tsx [hunk 3]
```diff
@@ -13,16 +13,27 @@ export default function Projects(): JSX.Element {
   const { t } = useTranslation(dictionary);
 
   return (
-    <div id="page-projects" className="theme-dark">
-      <header>
-        <h1 className="section-header-headline">{t('what')}</h1>
+    <div className="bg-[#000] text-white">
```
Comment (andrnors) [inline web/src/pages/projects/index.tsx:16]: --color-black

File: web/src/pages/_app.tsx [hunk 4]
```diff
@@ -90,10 +96,35 @@ function Main(): JSX.Element {
...
+        } lg:visible invisible flex justify-between items-center`}
+        id="menu-bar"
+      >
+        <Link href="/" className="leading-none pl-8">
+          <div id="menu-bar-logo" />
+          {/* <img className="logo" src="/static/logo.svg" alt="Logo" width="240" height="30" /> */}
+        </Link>
+        <div className="flex gap-x-8 pr-8">
+          <Link href="/people" className="no-underline leading-none nav-link">
+            <span className="text-xl text-[#000] hover:text-[#666]">{t('our people')}</span>
```
Comment (andrnors) [inline web/src/pages/_app.tsx:111]: all the colors 🔵 🟢 ⚫️ 🔴

File: web/src/pages/index.tsx [hunk 5]
```diff
@@ -46,67 +46,92 @@ function Home({ people }): JSX.Element {
...
-          <header className="people-header">
-            <h2 className="section-header-headline">{t('people')}</h2>
-            <p>{t('people_description', people.length)}</p>
-            <p>
-              <Button appearance={Button.appearances.LightNoPadding} href="/people">
-                {t('who_button')}
+      <main className="main">
+        <section className={`grid grid-cols-1 lg:grid-cols-2 `}>
+          <article className="relative col-span-1 lg:col-span-2 grid grid-cols-[1fr] lg:grid-cols-[repeat(2,1fr)] flex py-24 items-center gap-10">
+            <div className="full-screen-width h-full absolute bg-[#B6FF9E] -z-10"></div>
```
Comment (andrnors) [inline web/src/pages/index.tsx:56]: --color-green

File: web/src/styles/_page_home.css [hunk 6]
```diff
@@ -2,18 +2,27 @@
 ------------------------------------------------------------------------------*/
 #main-header {
   /* background: #e8e7e6 url("/static/seismograf.svg") no-repeat center; */
-  background-color: #e8e7e6;
+  background-color: #fff;
```
Comment (andrnors) [inline web/src/styles/_page_home.css:5]: add ```--color-white``` to _core.css and use var(--color-white)

**Review by andrnors — COMMENTED** [review]
Whoop! Nice!

**Review by myth — APPROVED** [review]

**Pull request merged**